image.yaml

```diff
@@ -73,17 +73,17 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_732_CR2
+                  ref: EAP_733_CR2
 
           - name: jboss-eap-image
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_732_CR2
+                  ref: EAP_733_CR2
 
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: 0.39.0
+                  ref: 0.42.0
```

Comment:
@drichtarik Unsure the _0.42.0_ version will work. But we can revert it back later, if necessary / it won't compile. We will see.

Reply:
Yes I am unsure about the necessity of upgrading this module when it comes to patches. This is a standard procedure but if i recall correctly we did not upgrade the version when we did 7.4.2